utility_analysis_new/dp_engine.py

```diff
@@ -39,7 +42,9 @@ def aggregate(self,
         result = super().aggregate(col, params, data_extractors,
                                    public_partitions)
         self._is_public_partitions = None
-        return result
+        return self._backend.map_values(
+            result, _compute_derived_metrics,
```

Comment:
Our goal is to compute aggregated metrics, WDYT about the following:
1.Implement combiner for computing aggregated metrics
2.Compute derived metrics in `create_accumulator`
3.Compute aggregated metrics with [combine_accumulators_per_key](https://github.com/OpenMined/PipelineDP/blob/52f03853698813adad3b6b749d62260bcfcba3c9/pipeline_dp/pipeline_backend.py#L106)